impl/src/main/java/org/jboss/arquillian/warp/impl/utils/Base64.java

```diff
@@ -457,6 +457,7 @@ void decode(byte[] in, int inPos, int inAvail) {
      *         <code>false</code>, otherwise
      * @deprecated 1.5 Use {@link #isBase64(byte[])}, will be removed in 2.0.
      */
+    @Deprecated
```

Comment:
OK this is fine, but next thing to do is remove the whole class.

Reply:
I created #273 - is this want you meant with your comment?

Reply:
@WolfgangHG That's exactly what I meant!